### DISCUSSION TITLE
Twill Form Builder

### DISCUSSION THREAD

**Discussion by MaartenJC** [area17/twill#1199] [Q&A] [answered]
I really like the concept of Twill and building my first website with it.
I was looking for a form builder to implement in my Twill CMS and stumbled upon https://form.twill.io

This looks very nice, but i can't find anything about this form builder.
Is there someone who has experience with creating a form builder in a Twill CMS who could give some tips?

**Comment by ifox** [comment, marked as answer]
Hey @mcouvreur! If you mean a form builder for your editors so they can compose your frontend form fields in different configurations, the best approach is most likely to leverage Twill's block editor. You could create a block per available form field, or a single more complex block with conditional fields, for example.

form.twill.io is a bit meta! It was a prototype of a form builder for Twill forms themselves. We sometimes use it to discuss how we and our client would like to organize a specific CMS form.

**Reply by MaartenJC** [reply]
Thanks for the quick response.
I agree that would work. I will go that route then.
